app/channels/application_cable/connection.rb

```diff
@@ -1,4 +1,19 @@
 module ApplicationCable
   class Connection < ActionCable::Connection::Base
+    identified_by :current_user
+
+    def connect
+      self.current_user = find_verified_user
+    end
+
+    private
+
+    def find_verified_user
+      if verified_user = User.find_by(id: cookies.encrypted[:user_id])
```

Comment:
Most of this is boilerplate code from [the docs](https://edgeguides.rubyonrails.org/action_cable_overview.html).

Here we're setting up a connection if we're able to authenticate this user, and rejecting it otherwise. Unfortunately, I can't figure out how to authenticate the user here. `current_user`, `cookies.encryped[:user_id]`, `cookies.signed[:user_id]`, and other variations are all nil.

The authentication/verification of the user is less important, but we do need to figure out which user is being connected to here. Any ideas @edance ?